vm/channel.go

```diff
@@ -138,8 +138,8 @@ func builtinChannelInstanceMethods() []*BuiltinMethodObject {
 			// @return [Null]
 			Name: "close",
 			Fn: func(receiver Object, sourceLine int, t *Thread, args []Object, blockFrame *normalCallFrame) Object {
-				if len(args) != 0 {
-					return t.vm.InitErrorObject(errors.ArgumentError, sourceLine, errors.WrongNumberOfArgument, 0, len(args))
+				if e, aLen := 0, len(args); e != aLen {
```

Comment:
I think it's not necessary to assign `0` to a variable, and it's not straightforward at first glance

Reply:
Actually I wondered this assignment: this was just for reducing repetitive `0` literals passing to InitErrorObject.
I'm OK to remove them.

Reply:
I reverted the scoped variables.